options_compile.cpp

```diff
@@ -287,10 +287,11 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
 
   auto parseClExt = [&](const std::string &clExtStr) {
     llvm::StringRef clExtRef(clExtStr);
-    clExtRef.consume_front("-cl-ext=");
+    assert(clExtRef.consume_front("-cl-ext=") &&
```

Comment:
"clExtRef.consume_front("-cl-ext=")" needs to be moved out of assert, because assert is noop in release mode

Reply:
done, thank you!